mag-part-1/curry/curry_exercises.js

```diff
@@ -6,25 +6,41 @@ var _ = require('ramda');
 //==============
 // Refactor to remove all arguments by partially applying the function
 
-var words = function(str) {
-  return split(' ', str);
-};
+// var words = function(str) {
+//   return _.split(' ', str);
+// };
+
+function splitHere(separator,str) {
+  return split(separator, str);
+}
+
+var words = _.partial(splitHere, ' ')
 
 // Exercise 1a
 //==============
 // Use map to make a new words fn that works on an array of strings.
 
-var sentences = undefined;
-
+var sentences = (arrayOfStrings) => {
+  return arrayOfStrings.map( function (element) {
+    return element.split(' ')
+  })
+}
 
 // Exercise 2
 //==============
 // Refactor to remove all arguments by partially applying the functions
 
-var filterQs = function(xs) {
+//var filterQs = function(xs) {
+//   return _.filter(function(x) {
+//     return match(/q/i, x);
+//   }, xs);
+// };
+
+function boobooFunction (xs) {
   return filter(function(x){ return match(/q/i, x);  }, xs);
 };
```

Comment:
I think they want xs removed from the function call so you could map or compose and pass in xs in the curried function and not as an argument of the parent function?